unity/Assets/Scripts/ContinuousMovement.cs

```diff
@@ -0,0 +1,194 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityStandardAssets.Characters.FirstPerson;
+using System;
+
+
+namespace UnityStandardAssets.Characters.FirstPerson {
+
+    public class ContinuousMovement {
+
+    public static void unrollSimulatePhysics(IEnumerator enumerator, float fixedDeltaTime) {
```

Comment:
Nice! This is what I was referring to when using the IEnumerable to unify the code.

Reply:
Yeah good idea, thanks for mentioning it